kolibri/core/assets/src/utils/contentNodeUtils.js

```diff
@@ -9,6 +9,9 @@ import { MasteryModelTypes } from 'kolibri.coreVue.vuex.constants';
  * from `files` is returned.
  */
 export function getContentNodeThumbnail(contentnode) {
+  if (!contentnode.thumbnail) {
```

Comment:
Thanks for the pull request! Unfortunately, this fix is too broad, and would cause the channel thumbnail to be used for every ContentNode that does not have a thumbnail. It will also be used everywhere this function is used, so it might be nicer to limit the scope of this to the Learn plugin.

I would suggest instead to make the fix in `ContentNodeThumbnail.vue`, check the return value of this function, and if it is falsy, then you can check whether the `contentNode` has a truthy `parent` attribute. If the `parent` attribute is falsy, that means it is a root node, and you can safely return the channel thumbnail.

Also, the `channel_thumbnail` property is not reliably defined, so you are better off using this function from the `useChannels` composable: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/composables/useChannels.js#L51

See here for an example of its use: https://github.com/learningequality/kolibri/blob/develop/kolibri/plugins/learn/assets/src/views/HybridLearningContentCard/index.vue#L113